src/pt/BreadCrumbs.jl

```diff
@@ -0,0 +1,62 @@
+"""
+A struct that provides a basic, user-friendly interface to Pigeons. Only two inputs
+are required, in positional order:
+$FIELDS
+
+!!! note
+
+    The PT state is initialized using a random sample from the reference.  
+"""
+struct BreadCrumbs{TRefDist <: Distributions.Distribution, TTarget}
+    """A function that evaluates the target log potential"""
+    target_log_potential::TTarget
+    """A Distributions.jl distribution used as reference"""
+    reference_distribution::TRefDist
+end
+
+# Target for a BreadCrumbs input
+struct BreadCrumbsTarget{TBC <: BreadCrumbs}
+    bc::TBC
+end
+function (bct::BreadCrumbsTarget)(x)
+    return if insupport(bct.bc.reference_distribution, x)
+        bct.bc.target_log_potential(x)
+    else
+        eltype(bct.bc.reference_distribution)(-Inf)
+    end
+end
+default_explorer(::BreadCrumbsTarget) = SliceSampler()
+
+# initialization
+# general case
+function initialization(bct::BreadCrumbsTarget, rng::AbstractRNG, ::Int)
+    rand(rng, bct.bc.reference_distribution)
+end
+# univariate case: need to wrap in vector to make the state mutable
+function initialization(
+    bct::TBCT,
```

Comment:
isn't this equivalent to `bct::Distributions.UnivariateDistribution`? etc for the other arguments